.eslintrc

```diff
@@ -0,0 +1,56 @@
+
```

Comment:
Why the empty line, and why does the indentation for the whole file pushed inside?

Reply:
I have removed the empty line and adjusted the indentation.